about.html

```diff
@@ -11,54 +11,59 @@
 
 </head>
 
+<header>
+  <nav>
+      <img src="/img/lambda-black.png" alt="logo">
+      <a href="/index.html">Home</a>
+      <a href="/about.html">About</a>
+      <a href="">Products</a>
+      <a href="">Blog</a>
+      <a href="">Contact</a>
+  </nav>
```

Comment:
Using a section is okay here, there are no rules regarding which of the semantic elements you use, a section is declared as a section of a document, In the general comments I will link an article and it shows a picture of how semantic elements are supposed to be used (at least one way to use them).